ethereum/erc/parsers.json

```diff
@@ -0,0 +1,145 @@
+                                }
+                            }
+                        }
+                    ],
+                    "functionName": "safeTransferFrom",
+                    "functionType": "transfer721",
+                    "options": {},
+                    "screen": [
+                        {
+                            "label": "Token Sender",
+                            "name": "from"
+                        },
+                        {
+                            "label": "Token Recipient",
+                            "name": "to"
+                        },
+                        {
+                            "label": "Token ID",
+                            "name": "tokenID"
+                        }
+                    ],
+                    "selector": "0x42842e0e"
+                }
+            ]
+        },
+        {
+            "address": "0x1155000000000000000000000000000000000000",
+            "name": "ERC-1155 Transfer",
+            "parsers": [
+                {
+                    "arguments": [
+                        {
+                            "format": "eip55",
+                            "name": "from",
+                            "value": {
+                                "address": {
+                                    "path": {
+                                        "default": "from"
+                                    }
+                                }
+                            }
+                        },
+                        {
+                            "format": "eip55",
+                            "name": "to",
+                            "value": {
+                                "address": {
+                                    "path": {
+                                        "default": "to"
+                                    }
+                                }
+                            }
+                        },
+                        {
+                            "format": "raw",
+                            "name": "tokenID",
+                            "value": {
+                                "uint256": {
+                                    "path": {
+                                        "default": "id"
+                                    }
+                                }
+                            }
+                        },
+                        {
+                            "format": "raw",
+                            "name": "amount",
+                            "value": {
+                                "uint256": {
+                                    "path": {
+                                        "default": "amount"
+                                    }
+                                }
+                            }
+                        }
+                    ],
+                    "functionName": "safeTransferFrom",
+                    "functionType": "transfer1155",
```

Comment:
transfer1155' is not one of ['stake', 'swap', 'wrap', 'unwrap', 'borrow', 'repay_loan', 'lend', 'withdraw', 'claim_reward', 'configure', 'buy', 'approve', 'read', 'transfer_erc721', 'transfer_erc1155']